cmd/apiserver/app/server/server.go

```diff
@@ -109,7 +109,7 @@ func NewCommandServer(out io.Writer) *cobra.Command {
 
 // runServer is a method on the options for composition. allows embedding in a
 // higher level options as we do the etcd and serving options.
-func (serverOptions ServiceCatalogServerOptions) runServer() error {
+func (serverOptions ServiceCatalogServerOptions) RunServer(stopCh <-chan struct{}) error {
```

Comment:
exported for the test and for reuse. Otherwise there is no way anyone can reuse this.

comment needs to be fixed to match the new name.